openrtb_ext/request_wrapper.go

```diff
@@ -285,14 +285,18 @@ func (rw *RequestWrapper) rebuildSourceExt() error {
 // ---------------------------------------------------------------
 
 type UserExt struct {
-	ext          map[string]json.RawMessage
-	extDirty     bool
-	consent      *string
-	consentDirty bool
-	prebid       *ExtUserPrebid
-	prebidDirty  bool
-	eids         *[]openrtb2.EID
-	eidsDirty    bool
+	ext                             map[string]json.RawMessage
+	extDirty                        bool
+	consent                         *string
+	consentDirty                    bool
+	prebid                          *ExtUserPrebid
+	prebidDirty                     bool
+	eids                            *[]openrtb2.EID
+	eidsDirty                       bool
+	consentedProvidersSettings      *ExtUserCPSettings
+	consentedProvidersSettingsDirty bool
+	consentedProviders              string
```

Comment:
Do you think it will be easier to handle `consentedProvidersSettings   *ExtUserCPSettings` only? Keep track of `consentedProviders` looks redundant. 
Look at `other fields - there is just one tracker for each object, no trackers for nested fields. What do you think?

Reply:
Good catch, modified to get rid of this unecessary field in `openrtb_ext/request_wrapper.go`

Reply:
Gus, I meant to keep track of object, not field.
```
consentedProvidersSettings      *ExtUserCPSettings -> leave this
	consentedProvidersSettingsDirty bool
	consentedProviders              string -> remove this
	consentedProvidersDirty         bool
```
In case you add more fields to the object - it will have everything to keep track of it without significant changes

Reply:
🤦‍♂️ 
I misread your comment.